docs/source/model_design/designing_vivarium_model/interactive_sim/index.rst

```diff
@@ -65,124 +65,167 @@ Another useful resource is the `Vivarium InteractiveContext documentation <https
 Example notebook
 ++++++++++++++++
 
-This `notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_sim_example_setup.ipynb>`_ includes all of the steps seen below using the CVD project as an example. 
+This `notebook <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/interactive_sim_example_setup.ipynb>`_ includes all of the steps seen below using the CVD project as an example.
+
+.. todo::
+  Update this example.
 
 .. _interactive_setup_1:
 
 Step 1: General Prep
 ++++++++++++++++++++
 
-  - Ensure that you have cloned the engineers' GitHub repo and have pulled the latest version of the simulation.  
-  - Check that you have `vivarium` installed in your environment and can import `InteractiveContext` in a Jupyter notebook. 
+  - Create a conda environment with the appropriate version of Python, if you haven't already.
+  - If you haven't installed the simulation in this environment yet, install it with :code:`pip install git+https://github.com/ihmeuw/<REPO>.git` or :code:`pip install git+https://github.com/ihmeuw/<REPO>.git@<BRANCH>` if you need a specific branch.
+  - If you have installed the simulation repository before and just want to update to the latest version or a different branch, run one of the above commands with :code:`--no-deps --force-install` added.
```

Comment:
Ooh this is good to know! I always just uninstall and reinstall the repo

Reply:
That works! It just sometimes updates other things, which you may or may not want.